docs/advanced-usage/normalizers.md

```diff
@@ -63,4 +63,6 @@ class ArrayableNormalizer implements Normalizer
 }
 ```
 
-Normalizers are executed the order as they are defined in the `normalize` method. The first normalizer not returning null will be used to normalize the payload. Magical creation methods always have precedence on normalizers.
```

Comment:
Let's keep these on one line, the spate docs site will make sure they're readable

Reply:
Done 💪